src/main/java/eu/lundegaard/liferay/db/setup/core/SetupFragments.java

```diff
@@ -54,7 +57,7 @@ public static void setupFragments(List<FragmentCollection> fragmentCollections,
                     createFragmentCollection(fragmentCollection, userId, groupId, serviceContext);
                     break;
                 case "update":
-                    updateFragmentCollection(fragmentCollection, groupId);
+                    updateFragmentCollection(fragmentCollection, userId, groupId, serviceContext);
```

Comment:
It looks like you switched userId and groupId method parameters. Look at the method declaration.